src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,58 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    //TODO: NEEDS REFACTORING TO THE BRIDGE PATTERN
+
+    private static final int EMPTY_STACK_HEAD_INDEX = -1;
+    private static final int FULL_STACK_HEAD_INDEX = 11;

```

Comment:
This constant is related to CAPACITY. Changing only one leads to an error (inconsistency).